.circleci/config.yml

```diff
@@ -475,13 +518,14 @@ workflows:
           <<: *not-on-integ-branch
       - build-linux-debian:
           name: build-with-redis-<<matrix.redis_version>>
-          <<: *on-integ-branch
+          <<: *on-integ-and-version-tags
           matrix:
             parameters:
               redis_version: ["6.0", "6.2"]
-      - platforms-build:
+      - build-platforms:
+          #@@<<: *on-integ-and-version-tags
+          <<: *on-any-branch
```

Comment:
Intentional, or just for the PR?

Reply:
This is diagnostic code, will remove it.